README.md

```diff
@@ -41,7 +41,7 @@ contributions welcome! :relaxed:
 
 ### __challenge 6__: break repeating-key xor
 - [Hamming Distance (Wikipedia)](https://en.wikipedia.org/wiki/Hamming_distance)
-- _Note: Step 4 of the process allows for a lot of experimentation, so if you aren't getting results play around with that step. Also try breaking something you encrypted yourself with your Challenge 5 code for testing._
+- _Note: Step 4 of the process allows for a lot of experimentation, so if you aren't getting results play around with that step. Normalized hamming distances are relatively small numbers so remember many keys could have extremely similar outputs. Maybe try breaking something you encrypted yourself with your Challenge 5 code for testing._
```

Comment:
Can you explain why this addition is helpful? Was "extremely similar outputs" of normalized hamming distances something that distracted you from the main problem when you were solving?

Reply:
I think it's helpful in the sense that if your ch6 isn't working, all of the existing docs funnel you towards the "is it english" scoring as the probable issue. I think adding this sentence broadens the suspects to also the hamming distance implementation and the potential to increase the number of key lengths that are attempted

Reply:
I think the existing docs do point in the right direction: step 4. Will close for now.